lib/task-data/tasks/node-sleep.js

```diff
@@ -0,0 +1,14 @@
+// Copyright 2017, Dell EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Node sleep',
+    injectableName: 'Task.Node.Sleep',
+    implementsTask: 'Task.Base.Linux.Commands',
+    options: {
+        duration: 10000, //in milliseconds
+        commands: ['sudo sleep $(awk "BEGIN{print {{options.duration}}/1000}")']
```

Comment:
@ytjohn 
This delay is configurable in secure, if you don't need delay, you can configure it to 0. You test reminds me we only have RackHD delay but don't have a real node delay, so I created this task that can also be used for other workflows.
Also after fixing secure erase job bug in:  https://github.com/RackHD/on-tasks/pull/416, You should be able achieve delay with noop task as you wanted.